src/main.py

```diff
@@ -280,15 +282,45 @@ def send_email(to_addr, subject, content):
     }
     return ses.send_email(**kwargs)
 
+def email_user__old_credentials_disabled(user_csvrow):
```

Comment:
two underscores, confusing?

Reply:
are you confused?

Reply:
Sorry, that was catty. I think it's perfectly clear.

Reply:
Double underscores in Python have special meaning for name mangling and/or magic methods like `__repr__`, that's why I classify an ordinary method using them as confusing (as you would if there was a method named `EmailUserOldCredentialsDisabled`; why it's using a class-name convention?)

Reply:
It's a function, not an 'ordinary method'. This isn't a problem to be 'fixed' but you can always open a PR